tests/view_tests/templates/jsi18n.html

```diff
@@ -4,6 +4,14 @@
 </head>
 
 <body>
+  <p id="formats">
```

Comment:
I'm unsure if a selenium test is needed. Would it be enough to test `get_formats()` or the context of the views using the test client, or is this approach adding some additional value?